project/ExplorerRay/README.md

```diff
@@ -0,0 +1,87 @@
+# onnxinfo - A tool to show ONNX model summary like torchinfo
+
+This project aims to develop a tool that provides a summary of ONNX models, similar to how torchinfo works for PyTorch models. The tool will focus on offering a layer-wise breakdown, MAC/parameter counts, input/output shapes, and other key model details, all accessible via C++ and Python APIs.
+
+## Basic Information
+
+`onnxinfo` provides a detailed summary of ONNX models, including layer structure and parameter insights, available through both C++ and Python interfaces. 
+
+GitHub URL: https://github.com/ExplorerRay/onnxinfo
+
+## Problem to Solve
+
+With the widespread use of ONNX as an open standard for machine learning models, users used to inspect the structure and parameters of these models by [Netron](https://github.com/lutzroeder/netron). However, it is not that convenient to use it when in a CLI-only environment. This project aims to provide a utility for summarizing ONNX models like torchinfo.
```

Comment:
Do you plan to compare the runtime performance with the existing tools (e.g., netron)?

Reply:
I think it is hard to compare because it uses GUI and requires browser to show result. However, I plan to show things in CLI just like torchinfo. What's more, most tools that I have seen so far are not for ONNX, so I think it is a trouble to find another tool to compare.

All in all, I didn't plan to compare with other tools now.

Reply:
Thanks.  Please be clear that you do not plan to compare runtime with other tools.

You will need to find a way to verify and benchmark without the comparison.  Please describe how you plan to do it in the proposal.

Reply:
Oh, you are talking about something about validating my tool, is it? I will use torchinfo output to check whether my output is right, but my focus in not runtime.

Reply:
No, I was talking about runtime performance.  But you should validate the correctness and include it in automatic testing too.

Reply:
I am not familiar with doing a runtime performance benchmark especially when there aren't other tools to compare. If there is another similar tool, I can use the same script and model to check performance. But if there aren't, I am not sure how to define what is fast and what is slow. Do you have any suggestions? Thanks.

Reply:
> Thanks. Please be clear that you do not plan to compare runtime with other tools.

If you do not want to compare runtime performance, just make it clear in the proposal.  I am not asking you to do it.  I am asking you to make it clear.